avocado/utils/vfio.py

```
@@ -0,0 +1,182 @@
+    :type count: int
+    :return: True if supported, False otherwise
+    :rtype: bool
+    """
+    argsz = struct.calcsize("IIII")
+    index = VFIO_PCI_MSIX_IRQ_INDEX
+    irq_info_request = struct.pack("IIII", argsz, 1, index, 1)
+    irq_info_response = ioctl(device_fd, VFIO_DEVICE_GET_IRQ_INFO, irq_info_request)
+    nirq = (struct.unpack("IIII", irq_info_response))[3]
+    if nirq < count:
+        return False
+    return True
```

Ruff (0.12.2)

Return the condition `not nirq < count` directly

Replace with `return not nirq < count`

(SIM103)